Makefile.depends

```diff
@@ -0,0 +1,40 @@
+# autogenerated
+$(call depends,base/all): | .install/base/db .install/base/settings .install/modules/meta.analysis .install/base/logger .install/base/utils .install/modules/uncertainty .install/modules/data.atmosphere .install/modules/data.land .install/modules/data.remote .install/modules/assim.batch .install/modules/emulator .install/modules/priors .install/modules/benchmark .install/base/remote .install/base/workflow
+$(call depends,base/db): | .install/base/logger .install/base/remote .install/base/utils
+$(call depends,base/logger): |
```

Comment:
This empty prerequisite list won't hurt anything, will it? I can't think of a reason it would, but would also alway delete it if writing by hand.

Reply:
ran it locally and it seems to work.